identity/client/src/pages/groups/detail/roles/index.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import { FC } from "react";
+import { C3EmptyState } from "@camunda/camunda-composite-components";
+import { TrashCan } from "@carbon/react/icons";
+import useTranslate from "src/utility/localization";
+import { useApi } from "src/utility/api/hooks";
+import { getRolesByGroupId } from "src/utility/api/groups";
+import EntityList from "src/components/entityList";
+import { useEntityModal } from "src/components/modal";
+import DeleteModal from "src/pages/groups/detail/roles/DeleteModal";
+import AssignRolesModal from "src/pages/groups/detail/roles/AssignRolesModal";
+
+type RolesProps = {
+  groupId: string;
+};
+
+const Roles: FC<RolesProps> = ({ groupId }) => {
+  const { t } = useTranslate("groups");
+
+  const {
+    data: roles,
+    loading,
+    success,
+    reload,
+  } = useApi(getRolesByGroupId, {
+    groupId: groupId,
+  });
+
+  const areNoRolesAssigned = !roles || roles.items?.length === 0;
```

Comment:
Negative naming for booleans is generally discouraged because it can lead to readability and cognitive load issues (if we later do !areNoRolesAssigned for example).
We can either flip it
```
  const areRolesAssigned = roles && roles.items?.length > 0;
```

or called something else 

```
  const isRolesListEmpty = !roles || roles.items?.length === 0;
```

Reply:
Makes sense, I based it on one of the other pages, so there were other instances of this, since it's a small change with no code impact (just code style) I just fixed all instances already to avoid a new issue.